Index: src/main/java/com/company/classes/Pachyderm.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.company.classes;\r\n\r\nimport com.company.classes.Animal;\r\n\r\nimport java.util.Random;\r\n\r\npublic class Pachyderm  extends Animal {\r\n    public void roam(){\r\n        Random rand = new Random();\r\n        int probability = rand.nextInt(100);\r\n\r\n        if(probability < 25){\r\n            System.out.println(\"Pachyderm decided to charge\");\r\n        }\r\n        else{\r\n            System.out.println(\"Pachyderm decided to roam around\");\r\n        }\r\n    }\r\n}\r\n
===================================================================
--- src/main/java/com/company/classes/Pachyderm.java	(revision 30268fc54c06fc03421e72032d87ac45e171d5d2)
+++ src/main/java/com/company/classes/Pachyderm.java	(date 1599712823162)
@@ -16,4 +16,5 @@
             System.out.println("Pachyderm decided to roam around");
         }
     }
+    public void sleep(){};
 }
Index: src/main/java/com/company/classes/Feline.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.company.classes;\r\n\r\nimport java.util.Random;\r\n\r\npublic class Feline extends Animal {\r\n    public void onCall() {\r\n        Random rand = new Random();\r\n        int probability = rand.nextInt(9);\r\n\r\n        if(probability < 3) {\r\n            this.makeNoise();\r\n        }\r\n\r\n        if(probability >= 3 && probability < 6 ) {\r\n            this.roam();\r\n        }\r\n\r\n        if(probability >= 6) {\r\n            this.sleep();\r\n        }\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/company/classes/Feline.java	(revision 30268fc54c06fc03421e72032d87ac45e171d5d2)
+++ src/main/java/com/company/classes/Feline.java	(date 1599710497152)
@@ -3,7 +3,7 @@
 import java.util.Random;
 
 public class Feline extends Animal {
-    public void onCall() {
+    public void sleep() {
         Random rand = new Random();
         int probability = rand.nextInt(9);
 
@@ -19,4 +19,6 @@
             this.sleep();
         }
     }
+
+    public void roam(){};
 }
