Index: src/main/java/com/company/classes/Animal.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.company.classes;\r\n\r\npublic abstract class Animal {\r\n    public void wakeUp() {\r\n        System.out.println(\"Default Wake up\");\r\n    };\r\n    public void makeNoise() { System.out.println(\"Default Make Noise\"); };\r\n    public void eat() { System.out.println(\"Default Eat\"); };\r\n    public void roam() { System.out.println(\"Default Roam\"); };\r\n    public void sleep() {\r\n        System.out.println(\"Default Sleep\");\r\n    };\r\n    public void printInfo() {\r\n        System.out.println(\"Default Print Info\");\r\n    };\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/company/classes/Animal.java	(revision 30268fc54c06fc03421e72032d87ac45e171d5d2)
+++ src/main/java/com/company/classes/Animal.java	(date 1599708996012)
@@ -6,10 +6,8 @@
     };
     public void makeNoise() { System.out.println("Default Make Noise"); };
     public void eat() { System.out.println("Default Eat"); };
-    public void roam() { System.out.println("Default Roam"); };
-    public void sleep() {
-        System.out.println("Default Sleep");
-    };
+    abstract void roam();
+    abstract void sleep();
     public void printInfo() {
         System.out.println("Default Print Info");
     };
